R/module_teal.R

```diff
@@ -162,10 +162,7 @@ srv_teal <- function(id, modules, raw_data, filter = teal_slices()) {
 
     # loading the data -----
     env <- environment()
-    datasets_reactive <- reactive({
-      if (is.null(raw_data())) {
-        return(NULL)
-      }
+    datasets_reactive <- eventReactive(raw_data(), ignoreNULL = TRUE, {
```

Comment:
Any reason for explicitly setting `ignoreNULL` to default?

Reply:
No, I should remove it because eventReactive has `ignoreNULL = TRUE` by default, which is exactly what we need here.